packages/auth/src/model/public_types.ts

```diff
@@ -280,6 +280,12 @@ export interface Auth {
     callback: (user: User | null) => void | Promise<void>,
     onAbort?: () => void
   ): Unsubscribe;
+
+  onFirebaseTokenChanged(
```

Comment:
q: When do we call `onFirebaseTokenChanged`? 

- Please add jsdoc 
- Do we need to implement a onFirebaseTokenChanged function in packages/auth/src/core/index.ts ?